platform/darwin/docs/theme/templates/task.mustache

```diff
@@ -80,10 +78,10 @@
           </div>
           {{/return}}
           {{#github_token_url}}
-          <div class="slightly-smaller show-on-github">
+          <div class="show-on-github">
             <a href="{{{github_token_url}}}">
-              show on github
-              <img class="show-on-github-icon" src="{{{path_to_root}}}img/github.svg"/>
+              <img style="margin-bottom: 1px;" class="show-on-github-icon" src="{{{path_to_root}}}img/github.svg"/>
+              View Source on Github
```

Comment:
Since this text is no longer all in lowercase, please capitalize “Github” as “GitHub”.